src/ci/docker/scripts/emscripten.sh

```diff
@@ -20,5 +20,5 @@ exit 1
 
 git clone https://github.com/emscripten-core/emsdk.git /emsdk-portable
 cd /emsdk-portable
-hide_output ./emsdk install 1.39.20
-./emsdk activate 1.39.20
+hide_output ./emsdk install 2.0.17
```

Comment:
Similarly this seems like it ought to be its own PR. I'm not qualified to comment on what version of emscripten we want to target our support towards, but general rule of thumb has been so far to use a low version to ensure that we support a wide variety of versions and don't accidentally depend on very new features.